### PR TITLE
【PPSCI Doc No.107】update doc for `ppsci.visualize.save_vtu_to_mesh`

### PR DESCRIPTION
**Pull request by ooooo-create**
### PR types

Others
### PR changes

Docs
### Describe

No.107
link #686

### CODE REVIEW

File: ppsci/visualize/vtu.py [hunk 1]
```diff
@@ -140,6 +140,21 @@ def save_vtu_to_mesh(
...
+        >>> data_dict = {
+        ...     "x": np.array([[1], [2], [3],[4]]),
+        ...     "y": np.array([[2], [3], [4],[4]]),
+        ...     "z": np.array([[3], [4], [5],[4]]),
+        ...     "u": np.array([[4], [5], [6],[4]]),
+        ...     "v": np.array([[5], [6], [7],[4]]),
+        ... }
+        >>> coord_keys = ("x","y","z")
+        >>> value_keys = ("u","v")
+        >>> ppsci.visualize.save_vtu_to_mesh(filename, data_dict, coord_keys, value_keys) # doctest: +SKIP
```
Comment (HydrogenSulfate) [inline ppsci/visualize/vtu.py:157]: 第163行改为:`tuple(data_dict[key] for key in coord_keys)`，否则会有警告`sys:1: FutureWarning: arrays to stack must be passed as a "sequence" type such as list or tuple. Support for non-sequence iterables such as generators is deprecated as of NumPy 1.16 and will raise an error in the future.`

**Review by HydrogenSulfate — CHANGES_REQUESTED** [review]
辛苦修改一下

File: ppsci/visualize/vtu.py (hunk 1 above)
Comment (ooooo-create) [reply ppsci/visualize/vtu.py:157]: 好的～，已添加

**Review by HydrogenSulfate — APPROVED** [review]
LGTM

**Pull request merged**